dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -1701,8 +1701,8 @@ BEGIN
     LTEXT "Total:", -1, 29, 124, 53, 10
     LTEXT "", 7024, 90, 121, 150, 14, 0, WS_EX_STATICEDGE
     AUTOCHECKBOX "Pe&rmite calculatorului apelator specificarea unei adrese IP proprii", 7022, 20, 144, 231, 10, BS_TOP | BS_MULTILINE | WS_GROUP
-    DEFPUSHBUTTON "Con&firmă", 1, 133, 169, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 197, 169, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 133, 169, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 197, 169, 60, 14
```

Comment:
Accelerator collision &R vs.
 AUTOCHECKBOX "Pe&rmite calculatorului apelator specificarea unei adrese IP proprii", 7022, 20, 144, 231, 10, BS_TOP | BS_MULTILINE | WS_GROUP
Check your Romanian Windows to find out which to change.